integration/operator_test.go

```diff
@@ -244,3 +279,45 @@ func writeAWSResourceValues() error {
 
 	return nil
 }
+
+func numberOfWorkers(clusterName string) (int, error) {
+	cluster, err := f.AWSCluster(clusterName)
+	if err != nil {
+		return 0, microerror.Mask(err)
+	}
+
+	return len(cluster.Spec.AWS.Workers), nil
```

Comment:
Could you use `keyv2.WorkerCount` here?

Reply:
Sure, I've added `keyv2.MasterCount` too.